.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,10 +1,8 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0">
     <option name="myName" value="Project Default" />
-    <inspection_tool class="PyArgumentEqualDefaultInspection" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
-    <inspection_tool class="PyAugmentAssignmentInspection" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
     <inspection_tool class="RequiredAttributes" enabled="true" level="WARNING" enabled_by_default="true">
-      <option name="myAdditionalRequiredHtmlAttributes" value="theme,classes,width,history,location,match" />
+      <option name="myAdditionalRequiredHtmlAttributes" value="theme,classes,width" />
```

Comment:
i feel like this may be out of date?

Reply:
use the red one, not the green one. bad merge.